packages/domain/src/use-cases/lifecycle/ISessionPresenter.ts

```diff
@@ -0,0 +1,42 @@
+import { ProfileId } from '../../entities';
+import type { LogoutReason } from '../wallets';
+
+/**
+ * The identifier of a Profile
+ */
+export type ProfileIdentifier = {
+  id: ProfileId;
+  handle: string;
+};
+
+/**
+ * Framework-agnostic Wallet representation
+ */
+export type WalletData = {
+  address: string;
```

Comment:
nit: could use `EthereumAddress` from `shared-kernel` which we could brand at some point in time

Reply:
good call